.circleci/config.yml

```diff
@@ -101,12 +102,15 @@ references:
         LIBSODIUM_VERSION: "1.0.16"
       # source: https://github.com/aeternity/docker-builder/blob/master/Dockerfile#L23
       command: |
-        LIBSODIUM_DOWNLOAD_URL="https://github.com/jedisct1/libsodium/releases/download/${LIBSODIUM_VERSION}/libsodium-${LIBSODIUM_VERSION}.tar.gz" \
-          && curl -fsSL -o libsodium-src.tar.gz "$LIBSODIUM_DOWNLOAD_URL" \
-          && mkdir libsodium-src \
-          && tar -zxf libsodium-src.tar.gz -C libsodium-src --strip-components=1 \
-          && cd libsodium-src \
-          && ./configure && make -j$(nproc) && sudo make install && sudo ldconfig
+        LIBSODIUM_DOWNLOAD_URL="https://github.com/jedisct1/libsodium/releases/download/${LIBSODIUM_VERSION}/libsodium-${LIBSODIUM_VERSION}.tar.gz"
+          ./ci/try curl -fsSL -o libsodium-src.tar.gz "$LIBSODIUM_DOWNLOAD_URL"
+          mkdir libsodium-src
```

Comment:
I see you removed `&&` which means that the command will not check if the previous one failed and will execute it. Is this on purpose?

Reply:
These commands are execute with `/bin/bash -eo pipefail` so any command which exits with >0 should also make the job fail.

Reply:
That makes sense. Does the try script return an exit code when failed?

Reply:
Now it does.